i18n/en.json

```diff
@@ -1,28 +1,30 @@
 {
     "@metadata": {},
-    "title": "Wikimedia OCR",
-    "image-url": "Image URL:",
-    "image-url-help": "This must start with one of the following domain names and be a full URL to an actual image file: $1",
+    "title": "WikimediaOCR",
+    "subtitle": "Transcribe text from images",
+    "form-heading": "Transcribe an image",
+    "image-url": "Image URL",
+    "image-url-help": "Insert an image URL hosted on a Wikimedia server such as: $1",
     "image-url-error": "Image URL must begin with {{PLURAL:$1|the following domain name|one of the following domain names}} and end with a valid file extension: $2",
     "image-alt-text": "The original image",
-    "language-code": "Two-letter language code (optional):",
-    "language-code-help": "The ISO639 code of the language of the text in the image.",
-    "engine": "OCR engine:",
-    "engine-help": "Choose between the open-source Wikimedia-hosted Tesseract, and Google's Cloud Vision API.",
-    "engine-name-google": "Google",
-    "engine-name-tesseract": "Tesseract",
+    "language-code": "Language (optional)",
```

Comment:
Since multiple languages are accepted, should this be "Languages (optional)"?

Reply:
Done!